docs/en/testing/supplemental.md

```diff
@@ -496,15 +494,18 @@ import { assert } from 'chai';
 describe('Action', () => {
 	const fetchItems = stub();
 	it('can fetch data on button click', () => {
-		const h = harness(() => <Action fetchItems={fetchItems} />);
-		h.expect(() => (
+		const WrappedButton = wrap(Button);
+		const template = assertionTemplate(() => (
 			<div classes={[css.root]}>
-				<Button key="button" onClick={() => {}}>
+				<WrappedButton key="button" onClick={() => {}}>
 					Fetch
-				</Button>
+				</WrappedButton>
 			</div>
 		));
-		h.trigger('@button', 'onClick');
+		const r = renderer(() => <Action fetchItems={fetchItems} />);
+		r.expect(template);
+		r.property(WrappedButton, 'onClick');
```

Comment:
It might be worth mentioning in the documentation below how the `onClick` property is found in the renderer using the wrapped button. It's not immediately clear how the `property` method is using the `WrappedButton` to find the real `Button` on the rendered widget when the rendered widget isn't constructed using the wrapped widgets.

Reply:
That seems reasonable, I'll add something into the supplemental docs.

Reply:
@devpaul I have added:

> The test renderer uses the location of a wrapped test node in the expected tree to attempt to perform the requested action (either `r.property()` or `r.child()`) on the actual output of the widget under test. If the wrapped test node does not match the corresponding node in the actual output tree then no action will be performed and the assertion will report a failure.

To the `Wrapped Test Nodes` section.

Reply:
Thanks for adding this. It'll be a good reference!